<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>980</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="allColumnsShowFocus">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>业务</string>
    </property>
    <addaction name="action1"/>
    <addaction name="separator"/>
    <addaction name="action2"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>筛选</string>
    </property>
    <addaction name="action3"/>
    <addaction name="separator"/>
    <addaction name="action4"/>
    <addaction name="separator"/>
    <addaction name="actiony"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action0"/>
    <addaction name="separator"/>
    <addaction name="actionx"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查询</string>
    </property>
    <addaction name="actionp"/>
    <addaction name="separator"/>
    <addaction name="actionq"/>
   </widget>
   <addaction name="menu_4"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action1">
   <property name="text">
    <string>进货</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>售出/退货</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>按商品编号</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>按销售日期</string>
   </property>
  </action>
  <action name="action0">
   <property name="text">
    <string>打开资源文件 </string>
   </property>
  </action>
  <action name="actionx">
   <property name="text">
    <string>保存当前修改</string>
   </property>
  </action>
  <action name="actiony">
   <property name="text">
    <string>显示所有信息</string>
   </property>
  </action>
  <action name="actiona">
   <property name="text">
    <string>按商品编号</string>
   </property>
  </action>
  <action name="actionp">
   <property name="text">
    <string>商品库存</string>
   </property>
  </action>
  <action name="actionq">
   <property name="text">
    <string>商品毛利</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
